pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -239,9 +247,12 @@ def backend_info(self) -> BackendInfo:
         return self._backend_info
 
     @classmethod
-    def _get_backend_info(cls, backend: "_QiskIBMBackend") -> BackendInfo:
-        config = backend.configuration()
-        characterisation = process_characterisation(backend)
+    def _get_backend_info(
+        cls,
+        config: QasmBackendConfiguration,
+        props: Optional[BackendProperties],
+    ) -> BackendInfo:
```

Comment:
I know that this is not a public function, but could you maybe add a short comment on what this function is doing?

Reply:
Sure. Done.